src/main/java/jp/skypencil/errorprone/slf4j/DoNotUseStaticSlf4jLogger.java

```diff
@@ -0,0 +1,59 @@
+package jp.skypencil.errorprone.slf4j;
+
+import static com.google.errorprone.BugPattern.SeverityLevel.SUGGESTION;
+import static com.google.errorprone.matchers.Matchers.allOf;
+import static com.google.errorprone.matchers.Matchers.isField;
+import static com.google.errorprone.matchers.Matchers.isStatic;
+
+import com.google.auto.service.AutoService;
+import com.google.errorprone.BugPattern;
+import com.google.errorprone.VisitorState;
+import com.google.errorprone.bugpatterns.BugChecker;
+import com.google.errorprone.bugpatterns.BugChecker.VariableTreeMatcher;
+import com.google.errorprone.fixes.Fix;
+import com.google.errorprone.matchers.Description;
+import com.google.errorprone.matchers.Matcher;
+import com.sun.source.tree.ModifiersTree;
+import com.sun.source.tree.VariableTree;
+import java.util.HashSet;
+import java.util.Set;
+import javax.lang.model.element.Modifier;
+
+@BugPattern(
+    name = "Slf4jLoggerShouldBeNonStatic",
+    summary = "Do not use static Logger field, use non-static one instead",
+    tags = {"SLF4J"},
+    severity = SUGGESTION)
+@AutoService(BugChecker.class)
+public class DoNotUseStaticSlf4jLogger extends BugChecker implements VariableTreeMatcher {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Override the "equals" method in this class. ([squid:S2160](https://sonarcloud.io/organizations/kengotoda-github/rules#rule_key=squid%3AS2160))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=jp.skypencil.errorprone.slf4j%3Aerrorprone-slf4j&pullRequest=4&issues=AWRd1zxd1IJZDfsS72je&open=AWRd1zxd1IJZDfsS72je)

[ISSUE_KEY]: AWRd1zxd1IJZDfsS72je